base/src/main/kotlin/net/corda/v5/base/types/MemberX500Name.kt

```diff
@@ -10,7 +10,9 @@ import javax.naming.ldap.Rdn
 import javax.security.auth.x500.X500Principal
 
 /**
- * X.500 distinguished name data type customised to how Corda uses names. This restricts the attributes to those Corda
+ * X.500 distinguished name data type customised to how Corda uses names.
```

Comment:
```suggestion
 * X.500 distinguished name customised to how Corda uses names.
```

Reply:
I think that is more correct without your change - it is a data type for an X500 distinguished name.